web/src/components/LandingPageSlider.tsx

```diff
@@ -0,0 +1,176 @@
+import { LandingPageActionTile } from "@/components/LandingPageActionTile";
+import { useConfig } from "@/lib/data-hooks/useConfig";
+import { QUERIES, ROUTES, routeWithQuery } from "@/lib/domain-logic/routes";
+import analytics from "@/lib/utils/analytics";
+import { useRouter } from "next/router";
+import { ReactElement, useRef, useState } from "react";
+import Slider from "react-slick";
+import "slick-carousel/slick/slick-theme.css";
+import "slick-carousel/slick/slick.css";
+import { useMountEffect } from "../lib/utils/helpers";
+
+/* eslint-disable @typescript-eslint/no-explicit-any */
+function SampleNextArrow(props: any) {
+  const { onClick } = props;
```

Comment:
seems like an extra line to destructure when we could just reference via props.onClick. Is there a reason for the destructuring?